project/Yojahuang/README.md

```diff
@@ -0,0 +1,93 @@
+# R tree search engine
+
+A R tree search engine written by C++.
+
+## Basic Information
+
+There are many libraries which can solve spatial searching querys. But their performance may be bounded when facing larger scale of dataset and queries.
+
+R tree search engine is a C++ library that aims to provide an efficient and scalable spatial search engine which can serve as a building block of map application.
+
+## Problem to Solve
+
+When we talked about map application, often time we'll have queries like “return all buildings in this area”, “find 1000 closest gas stations to this point”, and returning results within milliseconds even when searching millions of objects.
+
+The goal of R tree search engine is to provide a high performance search engine to solve these classical problems like "K nearest neighbors" and "range query".
```

Comment:
Please include a word or two for how you will evaluation the performance of the tree.